jit/v2/JIT_v2.0.0.yaml

```diff
@@ -191,7 +253,13 @@ paths:
       operationId: omit-port-call
       summary: Omits a Port Call
       description: |
-        Provider `OMITTING` a **Port Call**
+        Provider `OMITTING` a **Port Call**. This is called whenever the provider has to omit a **Port Call**.
+
+        When a consumer reieves an OMIT - it is the responsibility to pass on this information to any secondary receivers the consumer has called in the past using the `isFYI=true` property while creating **Port Calls** also using `isFYI=true`.
+
+        It is the responsibility of the provider to also send an OMIT to all **Terminal Calls** linked to the `portCallID` and to **Cancel** all `PortCallServices` linked to omitted **Terminal Calls** because of this **Port Call** omit.
+
+            Once a **Port Call** has been OMITTED - it is **NOT** possible to undo. In case the OMIT has to be "undone" a new **Port Call** must be created with new **Terminal Calls** and new **Port Call Services**.
```

Comment:
Once a **Port Call** has been OMITTED, this action **CANNOT** be undone. In case the OMIT has to be "undone" a new **Port Call** must be created with new **Terminal Calls** and new **Port Call Services**.